test.js

```diff
@@ -1,9 +1,9 @@
-var ClassList = require('./')
-var hastscript = require('hastscript')
-var test = require('tape')
+import ClassList from './index.js'
+import { h as hastscript } from 'hastscript'
```

Comment:
here is breaking change in `hastscript` since [7.0.0](https://github.com/syntax-tree/hastscript/releases/tag/7.0.0) which also use ESM

Reply:
Should be okay, it's only a devDependency, and I think I'll probably have to do a major version bump anyway.